package.json

```diff
@@ -34,7 +34,7 @@
     "del": "~2.2.0",
     "edx-pattern-library": "~0.12.5",
```

Comment:
We generally don't update deps until we need to. Nothing visual has changed in the UITK for a while.